script.js

```diff
@@ -0,0 +1,63 @@
+const recipe1 = {
+  title: "Pancake",
+  ingredients: ["flour", "baking powder", "sugar", "salt", "milk", "egg"],
+  instructions:
+    "Step 1: Sift the dry ingredients together.\nStep 2: Make a well, then add the wet ingredients. Stir to combine\nStep 3: Scoop the batter onto a hot griddle or pan.\nStep 4:Cook for two to three minutes, then flip.\nStep 5:Continue cooking until brown on both sides. ",
+};
```

Comment:
Recipe objects should have id. I will suggest to follow the format of the example given in the exercise description :)